src/components/editor/EditorArticleController.jsx

```diff
@@ -56,7 +57,7 @@ export default class EditorArticleController extends FalcorController {
   }
 
   static getFalcorPathSets(params) {
-    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']]];
+    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']], ['latestIssue']];
```

Comment:
I'm thinking you could already think of this as an array though, where latestIssue is the call to the length of that array.

If we want to add a creation date to it though, then I can see the arrays relevance